src/taglibs/migrate/unless-tag.js

```diff
@@ -0,0 +1,39 @@
+const printJS = require("./util/printJS");
+const commonTagMigrator = require("./all-tags");
+
+module.exports = function migrator(oldNode, context) {
+    if (oldNode.tagName !== "unless") {
+        return;
+    }
+    const attributes = oldNode.attributes;
+    const argument = oldNode.argument;
+    commonTagMigrator(oldNode, context);
+    oldNode.setTransformerApplied(commonTagMigrator);
+
+    context.deprecate(
+        'The "<unless(x)>" tag is deprecated. Please use "<if(!x)>" instead. See: https://github.com/marko-js/marko/wiki/Deprecation:-unless-tag'
+    );
+    if (!argument) {
+        context.addError(
+            "Invalid <unless> tag. Argument is missing. Example; <unless(foo === true)>"
+        );
+        return oldNode;
+    }
+    if (attributes.length) {
+        context.addError("Invalid <unless> tag. Attributes not allowed.");
+        return oldNode;
+    }
+
+    const builder = context.builder;
+    const newNode = builder.htmlElement(
+        "if",
+        undefined,
+        [],
+        printJS(`!${argument}`, context),
```

Comment:
This line should switch to use `builder.negate` like so:

```js
printJS(builder.negate(argument), context),
``` 

This will fix cases like `<unless(x && y)>` which would currently become `<if(!x && y)>` (we should add a test case for that as well).